--- ./CMakeLists.old.txt	2025-05-05 18:50:55.648660062 -0400
+++ ./CMakeLists.txt	2025-05-05 18:51:42.545281707 -0400
@@ -221,15 +221,15 @@
 # Include application targets.
 # Comes from the <target>/CMakeLists.txt file in the current directory.
 # TODO: Change these lines to match your project target when you copy this file.
-if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/tests")
-  add_subdirectory(tests/cefsimple)
-  add_subdirectory(tests/gtest)
-  add_subdirectory(tests/ceftests)
-endif()
-
-if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/tests/cefclient")
-  add_subdirectory(tests/cefclient)
-endif()
+# if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/tests")
+#   add_subdirectory(tests/cefsimple)
+#   add_subdirectory(tests/gtest)
+#   add_subdirectory(tests/ceftests)
+# endif()
+#
+# if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/tests/cefclient")
+#   add_subdirectory(tests/cefclient)
+# endif()
 
 # Display configuration settings.
 PRINT_CEF_CONFIG()
